qt/src/OutputEditorHOCR.cc

```diff
@@ -1223,11 +1225,18 @@ void OutputEditorHOCR::savePDF() {
 	if(!failed.isEmpty()) {
 		QMessageBox::warning(m_widget, _("Errors occurred"), _("The following pages could not be rendered:\n%1").arg(failed.join("\n")));
 	}
+
+    bool pdfCanBeOpened = true;
```

Comment:
Can you configure your editor to use tabs for indentation?

Reply:
Is it problem?

Reply:
Well, yes, you typically try to not mix indentation stiles within a project, it makes the source files very un-readable.